index.html

```diff
@@ -44,19 +44,32 @@ <h1><i class="snes-jp-logo brand"></i>NES.css</h1>
 
     <section class="nes-container with-title">
       <h2 class="title">Buttons</h2>
-      <div>
+      <div id="buttons">
         <button type="button" class="nes-btn">Normal</button>
         <button type="button" class="nes-btn is-primary">Primary</button>
         <button type="button" class="nes-btn is-success">Success</button>
         <button type="button" class="nes-btn is-warning">Warning</button>
         <button type="button" class="nes-btn is-error">Error</button>
         <button type="button" class="nes-btn is-disabled">Disabled</button>
       </div>
+      <button type="button" class="nes-btn is-error show-code" id="show-buttons-code"><></button>
+      <span class="nes-balloon from-right copied" id="copied-buttons">Copied!</span>
+    </section>
+    <section class="nes-container code" id="code-buttons">
+      <code>
```

Comment:
The Preformatted Text element is better here ☝️ 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/pre
```html
<pre><code>
...
</code></pre>
```

Reply:
Okay :+1: 
But without this:

```css
pre {
  white-space: normal;
}
```

Looks weird, too spaced, ok?